install/kubernetes/helm/istio-init/files/crd-10.yaml

```diff
@@ -686,6 +686,29 @@ spec:
   scope: Namespaced
   version: v1alpha2
 ---
+kind: CustomResourceDefinition
```

Comment:
should this be `crd-1.1.yaml` instead?  Or is `crd-10.yaml` the place for all mixer CRDs ?

Reply:
We still haven't fully removed the `--useAdapterCRDs` option (for 1.1, I'm not even sure we changed the default :( )

Reply:
@sdake Is crd-1.1 the right place to put new CRD?

Reply:
@sdake Any ideas?